demo/cli/src/lib.rs

```diff
@@ -211,6 +211,12 @@ pub fn run<I, T>(args: I) -> error::Result<()> where
 		timestamp: Some(TimestampConfig {
 			period: 5,					// 5 second block time.
 		}),
+		treasury:  Some(TreasuryConfig {
+			proposal_bond: Permill::from_percent(5),
+			proposal_bond_minimum: 1_000_000,
+			spend_period: 12 * 60 * 24,
```

Comment:
hm, this looks like a timestamp or something, but it's actually should be a block number

Reply:
it should be a block number *per se* but rather a number of blocks for which the type is `BlockNumber`. in this case, we assume 12 blocks per minute (1 every 5 seconds), multiply by 60 and 24 to get approximately one day.